project/config/src/config/core.extension.yml

```diff
@@ -39,6 +38,8 @@ module:
   entity_reference_revisions: 0
   facets: 0
   facets_pretty_paths: 0
+  facets_summary: 0
+  fastly: 0
```

Comment:
We should only install fastly on production.

Reply:
Thanks , I have now updated this